keyboards/bastardkb/dilemma/4x6_4/info.json

```diff
@@ -195,10 +194,10 @@
             {"matrix": [8, 3], "x": 175, "y": 38, "flags": 4},
             {"matrix": [8, 4], "x": 162, "y": 40, "flags": 4},
             {"matrix": [8, 5], "x": 149, "y": 42, "flags": 4},
-            {"matrix": [9, 3], "x": 170, "y": 51, "flags": 4},
-            {"matrix": [9, 1], "x": 156, "y": 53, "flags": 4},
-            {"matrix": [9, 2], "x": 143, "y": 57, "flags": 4},
-            {"matrix": [9, 4], "x": 132, "y": 64, "flags": 4}
+            {"matrix": [9, 4], "x": 170, "y": 51, "flags": 4},
+            {"matrix": [9, 2], "x": 156, "y": 53, "flags": 4},
+            {"matrix": [9, 1], "x": 143, "y": 57, "flags": 4},
+            {"matrix": [9, 3], "x": 132, "y": 64, "flags": 4}
         ]
```

Comment:
I think this might be reverting a more recent change to correct a mistake I made in the RGB matrix coordinates for the initial Dilemma Max PR